website_event_snippet_calendar/static/src/xml/snippets.xml

```diff
@@ -1,14 +1,20 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <!-- Copyright 2018 Tecnativa - Jairo Llopis
      License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
-<template>
+<templates>
     <t t-name="website_event_snippet_calendar.list">
         <t t-foreach="events" t-as="event">
             <div class="row">
                 <div class="col-12 event_details">
                     <span class="date">
                         <t t-esc="event.date_begin" />
                     </span>
+                    <span>
+                        <i class="fas fa fa-arrow-right text-muted mx-1 my-1" />
+                    </span>
+                    <span class="date">
+                        <t t-esc="event.date_end" />
```

Comment:
Could we make it optional? Maybe some users don't want to show them both

Reply:
ok. Optional you mean exactly where and how you want to show it?

Reply:
Well, more whether or not you should show it

Reply:
@chienandalu, you can review the changes, to make it optional for users to show/hide the end date of events.